### ISSUE TITLE
How to Change the One by One Cloze Colour for Anking Overhaul?

### ISSUE THREAD

**Issue by GuruHokage** [AnKing-VIP/ankiweb-placeholder#73]
#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [x] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [x] I've checked if anyone else asked this question before by looking through the issue reports.

#### What Anki version are you using? ⁨

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
2.1.56 (haven't upgraded to the most recent yet - waiting until my exam is over)
```

#### I've been using the ANKing Note Type Easy Customization package. Everything mostly works great and I've been really happy with the variety it offers and the ease to customize as I don't really know much about coding. The one thing I can't figure out is how to change the font colour on the one by one CLOZE when I'm using it through the ANKing Overhaul option. The default colour is this dark green which doesn't show up well on my dark background. I can't seem to find the option in the "colour" category to change it. I've tried toggling various colour and none of them work. How do I change it? 

Also, I'm not sure if there's a better way to use the one by one in Anking Overhaul. If I copy/paste my Text Note into the one by one section on the "Add" page, it won't actually let me add the card (it says something like cloze is not supported). However if I go to Browse or Edit after I've made the card and copy/paste it into the One by One category, the same message pops up but the card has already been added and when I close it, the one by one works. Is there a better way to do this? 

Thanks so much!! Really appreciate all the effort you put into this! 

*A clear and concise question about the add-on.*

**Comment by GuruHokage** [comment]
I figured it out finally! For anyone else who has been trying to change this hopefully this helps! 

To change the colours you need to click on the "card" option under the edit or browse section. Click on the "Back Template" and then you have to scroll about half way through (quite a ways down) until you find the "Cloze one by one script" which is after "Auto Flip Back Script". Under the "Cloze one by one script" you have the option to change the cloze one by one colour and the cloze one by one hint colour. 

<img width="671" alt="Screen Shot 2023-11-23 at 1 23 52 PM" src="https://github.com/AnKing-VIP/ankiweb-placeholder/assets/151873031/e9ba5c55-3a8c-4ee0-bf9f-71081d323061">

**Comment by AnKingMed** [comment]
@abdnh @RisingOrange we've gotten this question a reasonable amount lately. Do you think we could move the color part at least to the style and maybe add it to the add-on?

**Comment by abdnh** [comment]
Will take a look

**Comment by abdnh** [comment]
Done. I added it to the add-on too.

**Comment by AnKingMed** [comment]
Awesome! @abdnh can you push the add-on with the new updated note types to ankiweb?  Can include this update if we're planning to include this for everyone? https://github.com/AnKing-VIP/AnKing-Note-Types/issues/146

**Comment by abdnh** [comment]
Update pushed.

**Comment by GuruHokage** [comment]
Thanks for the update! Unfortunately, I accidentally clicked on the update thinking that I would be able to keep whatever setting/colours I had already set the different cloze vs. one-by-one cloze to, which doesn't seem to be the case. With the previous settings, whatever the one-by-one cloze colour was seemed to over-ride the general cloze (or at least night mode Cloze). In this case, the general cloze (night mode) seems to over-ride the one-by-one cloze so even if I go in and change the one-by-one cloze colour (including the hint colour), it's showing up as my night mode cloze colour choice. Only when I change the general night mode Cloze colour does the one-by-one cloze colour actually change to my colour choice. 

This is not ideal as now I don't have a way to distinguish between a one-by-one cloze card vs. a regular cloze card which I had actually really liked and was using quite a bit! 

Is there a way to make it so that the update works the same as the previous version where the one-by-one Cloze colour would over-ride the night cloze and general cloze colour? Thank you again for making the update in the first place though!

**Comment by AnKingMed** [comment]
That's how it's supposed to work. @abdnh do we need to add a !important or something?

**Comment by GuruHokage** [comment]
There was always the option to change the one-by-one cloze type under the "Style" but changing that colour in the configuration (under both general and Anking) never actually changed the colour when the one-by-one was used. Instead, it would always default to the dark green colour. Previously, by using the workaround that I figured out above, I could change both the front bracket colour (is that Hint colour?) and the back font colour. 

With the new update, that option to change the colour using the work around I found has been completely taken away (I can't find that same colour configuration area) while the colour option under the "style" section remains. But now, instead of defaulting to the dark green colour, it defaults to the Night Mode Cloze colour (while I'm in Night Mode) which doesn't help b/c now all my clozes look exactly the same and I can't distinguish between the two. 

I did notice that there isn't a one-by-one cloze colour option for both hint or font colour under night mode. I have my anki set to night mode preferentially as I like the darker colour contrast better. Is there a way to add that option under night mode as well?
